package.nls.json

```diff
@@ -4,6 +4,9 @@
     "AWS.title.creatingCredentialProfile": "Saving new credential profile {0}",
     "AWS.title.selectCredentialProfile": "Select an AWS credential profile",
     "AWS.cdk.explorerTitle": "Explorer",
+    "Aws.cdk.explorerNode.noApps": "[no apps]",
```

Comment:
This could be slightly more explanatory, and should be in title casing. Consider:
`[No CDK Apps found in Workspaces]`

Reply:
that makes sense. I initially started capitalizing them, but noticed that all of the current placeholder nodes (such as no stacks, no lambda functions) are all lower cased.

Should I be going for consistency wrt capitalization?

Reply:
Coincidentally I've been working on those node texts in mainline this past week :)
I'd suggest aiming for ideal casing and not what the existing node text looks like